docs/unified/novacustom/hw-compatibility.md

```diff
@@ -0,0 +1,62 @@
+# Hardware compatibility
+
+## Introduction
+
+This page contains the list of peripherals that have been verified to work with
+NovaCustom laptops by the Dasharo validation team. They are tested with each
+release for each laptop currently supported.
```

Comment:
@BeataZdunczyk @mkopec Let's make sure this is in sync with the latest communication.

Reply:
Updated in 93aa9595b91a800d939977382e0f8751f981a280